WORKSPACE

```diff
@@ -162,6 +162,53 @@ new_http_archive(
     sha256 = "8e387d459216500ed7f908b66e63dae629a7872bc72eafaa0cd8fb339da00730",
 )
 
+KOTLIN_BUILD_FILE_CONTENT = """
+java_import(
+  name = 'kotlin',
+  jars = ['kotlin/lib/kotlin-plugin.jar'],
+  visibility = ['//visibility:public'],
+)
+java_import(
+  name = 'integration-test-deps',
+  jars = [
+      'kotlin/lib/kotlinx-coroutines-core.jar',
+      'kotlin/lib/kotlinx-coroutines-jdk8.jar',
+      'kotlin/lib/kotlin-script-runtime.jar',
+      'kotlin/lib/kotlin-script-util.jar',
+      'kotlin/lib/sam-with-receiver-ide-plugin.jar',
+      'kotlin/lib/sam-with-receiver-compiler-plugin.jar',
+      'kotlin/lib/allopen-compiler-plugin.jar',
+      'kotlin/lib/allopen-ide-plugin.jar',
+      'kotlin/lib/noarg-compiler-plugin.jar',
+      'kotlin/lib/noarg-ide-plugin.jar',
+  ],
+  visibility = ['//visibility:public'],
+)
+"""
+
+# Kotlin plugin for IntelliJ CE 2017.2. Required at compile-time for kotlin-specific features.
+new_http_archive(
```

Comment:
The intellij-community itself contains the kotlin plugin under plugins/Kotlin/lib/*.jar, so we don't need these.

Reply:
@chaoren Ok I got it working with IJ. Currently I have created a `intellij_platform_sdk:kotlin` / `kotlin_for_test`. 

 I assume the Kotlin plugin is available in all variants (Clion, Android Studio and IJ), I'll add the jars to `intellij_platform_sdk:sdk` ?.

Reply:
It's not in CLion.

Reply:
I'm not going to make any assumptions and only include Kotlin in IJ as it extends to the sdk compat as well.